cmd/cmd.go

```diff
@@ -1,5 +1,20 @@
 package cmd
 
+import "io"
+
+var (
+	// BinaryName is the name of the app.
+	// By default this is exercism, but people
+	// are free to name this however they want.
+	// The usage examples and help strings should reflect
+	// the actual name of the binary.
+	BinaryName string
+	// Out is used to write to information.
+	Out io.Writer
+	// Err is used to write errors.
+	Err io.Writer
+)
+
```

Comment:
I moved these into the `cmd.go` file from the `root.go` file, since we're collecting shared logic here. I also deleted the unused `In` variable.

Reply:
Good call on the consolidation.